src/lib.rs

```diff
@@ -113,6 +113,8 @@
 //! * `fixed_point` - use fixed point arithmetic instead of floating point for all trigonometric
 //! calculation.
 //!
+//! * `defmt` - provide implementations of `defmt::Format` for all types where possible.
```

Comment:
A link to the the defmt would be useful for users who aren't familiar with defmt.

Reply:
:+1: